corehq/apps/locations/models.py

```diff
@@ -438,7 +438,8 @@ def __init__(self, *args, **kwargs):
             self.location_type = location_type
 
     def __repr__(self):
-        return "%s (%s)" % (self.name, self.location_type)
+        val = u"%s (%s)" % (self.name, self.location_type)
+        return val.encode('utf-8')
```

Comment:
Okay, nice, thanks.  I was worried we had something like `"Your location is {}".format(location)`, where the user would see the `__repr__`